install/update-docker-images.sh

```diff
@@ -1,12 +1,18 @@
 echo "${_group}Fetching and updating $CONTAINER_ENGINE images ..."
 
-# We tag locally built images with a '-self-hosted-local' suffix. `docker
-# compose pull` tries to pull these too and shows a 404 error on the console
-# which is confusing and unnecessary. To overcome this, we add the
-# stderr>stdout redirection below and pass it through grep, ignoring all lines
-# having this '-onpremise-local' suffix.
+if [ "$CONTAINER_ENGINE" = "docker" ]; then
```

Comment:
Wanna swap the order like the others here. Since our default is `docker` let's only check `if podman` case, `else` do the default thing.